wagtail/contrib/forms/templates/wagtailforms/index_submissions.html

```diff
@@ -97,7 +97,7 @@ <h1>
                                 {% include "wagtailadmin/shared/field_as_li.html" with field=field field_classes="field-small" li_classes="col4" %}
                             {% endfor %}
                             <li class="submit col2">
-                                <button name="action" value="filter" class="button button-filter">{% trans 'Filter' %}</button>
+                                <button type="button" name="action" value="filter" class="button button-filter">{% trans 'Filter' %}</button>
```

Comment:
I think this one does need to be `type="submit"`, since it's only on form submission / page reload that the filter will be applied to the listing.